installing invoker with ruby-2.2.0-preview1 fails

this is because of eventmachine.

compilation issues:

```
compiling binder.cpp
compiling cmain.cpp
compiling ed.cpp
compiling em.cpp
em.cpp:827:9: error: use of undeclared identifier 'rb_thread_select'; did you mean 'rb_thread_fd_select'?
        return EmSelect (maxsocket+1, &fdreads, &fdwrites, &fderrors, &tv);
               ^~~~~~~~
               rb_thread_fd_select
./em.h:25:20: note: expanded from macro 'EmSelect'
  #define EmSelect rb_thread_select
                   ^
/Users/yuva/.rvm/rubies/ruby-2.2.0-preview1/include/ruby-2.2.0/ruby/intern.h:439:5: note: 'rb_thread_fd_select' declared here
int rb_thread_fd_select(int, rb_fdset_t *, rb_fdset_t *, rb_fdset_t *, struct timeval *);
    ^
em.cpp:827:32: error: cannot initialize a parameter of type 'rb_fdset_t *' with an rvalue of type 'fd_set *'
        return EmSelect (maxsocket+1, &fdreads, &fdwrites, &fderrors, &tv);
                                      ^~~~~~~~
/Users/yuva/.rvm/rubies/ruby-2.2.0-preview1/include/ruby-2.2.0/ruby/intern.h:439:42: note: passing argument to parameter here
int rb_thread_fd_select(int, rb_fdset_t *, rb_fdset_t *, rb_fdset_t *, struct timeval *);
                                         ^
em.cpp:946:6: error: use of undeclared identifier 'rb_thread_select'; did you mean 'rb_thread_fd_select'?
                                        EmSelect (0, NULL, NULL, NULL, &tv);
                                        ^~~~~~~~
                                        rb_thread_fd_select
./em.h:25:20: note: expanded from macro 'EmSelect'
  #define EmSelect rb_thread_select
                   ^
/Users/yuva/.rvm/rubies/ruby-2.2.0-preview1/include/ruby-2.2.0/ruby/intern.h:439:5: note: 'rb_thread_fd_select' declared here
int rb_thread_fd_select(int, rb_fdset_t *, rb_fdset_t *, rb_fdset_t *, struct timeval *);
    ^
3 errors generated.
make: *** [em.o] Error 1

make failed, exit code 2
```

please check https://github.com/eventmachine/eventmachine/issues/509 and https://github.com/eventmachine/eventmachine/issues/495

--- a/invoker.gemspec
+++ b/invoker.gemspec
@@ -29,7 +29,7 @@ Gem::Specification.new do |s|
   s.add_dependency("rainbow", "~> 2.0")
   s.add_dependency("iniparse", "~> 1.1")
   s.add_dependency("formatador", "~> 0.2")
-  s.add_dependency("eventmachine", "~> 1.0")
+  s.add_dependency("eventmachine", "~> 1.0.4")
   s.add_dependency("em-proxy", "~> 0.1")
   s.add_dependency("rubydns", "~> 0.8.5")
   s.add_dependency("uuid", "~> 2.3")

